### PR TITLE
IPSRE-133 Reconfigured Core-Front to work across multiple dev environments

### PR DESCRIPTION
**Pull request by vam-gds**
### What changed
IPSRE-133 Reconfigured Core-Front to work across multiple dev environments

### Why did it change
IPSRE-133 Reconfigured Core-Front to work across multiple dev environments

### Issue tracking

- [IPSRE-133](https://govukverify.atlassian.net/browse/IPSRE-133)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[IPSRE-133]: https://govukverify.atlassian.net/browse/IPSRE-133?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/template.yaml [hunk 1]
```diff
@@ -35,8 +31,8 @@ Conditions:
   IsDevelopment: !Or
     - !Equals [ !Ref AWS::AccountId, "130355686670"]
     - !Equals [ !Ref AWS::AccountId, "175872367215"]
-  IsOldDevelopment: !Equals [ !Ref AWS::AccountId, "130355686670"]
-  IsNewDevelopment: !Equals [ !Ref AWS::AccountId, "175872367215"]
+  IsDev01: !Equals [ !Ref AWS::AccountId, "130355686670"]
+  IsDev02: !Equals [ !Ref AWS::AccountId, "175872367215"]
```
Comment (Joe-Edwards-GDS) [inline deploy/template.yaml:35]: FYI another property might need changing after my latest merge, as it's also used to control the `ANALYTICS_DOMAIN` (see: https://github.com/alphagov/di-ipv-core-front/pull/1051/files#r1317427584)

Let me know if you want to discuss :)

Comment (vam-gds) [reply deploy/template.yaml:35]: Hi Joe - you will need to change your commit as relevant - this allows for devs to deploy to either dev01 or dev02 dev accounts.

**Review by mrdoodles — APPROVED** [review]

File: deploy/template.yaml (hunk 1 above)
Comment (Joe-Edwards-GDS) [reply deploy/template.yaml:35]: Sorry, mine was just merged... you should see it if you rebase.

It looks like the domains will now be `xxx.01.dev.identity.account.gov.uk` or `xxx.02.dev.identity.account.gov.uk`? - in which case the analytics domain in dev will need to match (maybe we can pull out a variable or something for the domain to make it easier).

Let me know if I can help

Comment (TobySaundersGDS) [reply deploy/template.yaml:35]: @Joe-Edwards-GDS changes are to make the dev analytics domain follow the previously correct pattern, the .01 or .02 sub(subsub...?) domain will need to be included as well now

Comment (Joe-Edwards-GDS) [reply deploy/template.yaml:35]: Ok - do you want me to update this branch accordingly? I think you'll need to rebase to see my changes.

Comment (vam-gds) [reply deploy/template.yaml:35]: Joe I am going to merge this one in feel free to open this branch up again or use a different branch completely up to you - need this in to start testing dev deployments.

**Pull request merged**